### DISCUSSION TITLE
How to save all query on terminal?

### DISCUSSION THREAD

**Discussion by Oggy4th** [pocketbase/pocketbase#2513] [Q&A] [answered]
How to save all query on terminal?

![image](https://github.com/pocketbase/pocketbase/assets/44497354/a1e5b53b-f70c-4c4d-aebd-f89cfc97e1df)

**Comment by ganigeorgiev** [comment, marked as answer]
Could you elaborate a little more on your use case?

The printed queries are visible only when the `--debug` flag is set (_defaults to true when `go run` is used_) and note that the output may not be valid raw SQL string.

In general you can pipe the stdout/stderr output and write it in a file with `tee` (_the color characters may not be preserved_). Something like:


```go
go run main.go serve |& tee ./output.txt
```

**Reply by Oggy4th** [reply]
@ganigeorgiev 

using it like another backup plan.

You can find more details from here https://youtu.be/tLdRBsuvVKc

**Reply by ganigeorgiev** [reply]
As mentioned, the above queries are intended only for debug purpose.

If you want replication you can check [litestream](https://github.com/benbjohnson/litestream), [litefs](https://github.com/superfly/litefs), [marmot](https://github.com/maxpert/marmot), etc.

Additionally, the [latest v0.16.0](https://github.com/pocketbase/pocketbase/releases/tag/v0.16.0) comes with cron based auto backups.
